schema_diagram.sql

```diff
@@ -0,0 +1,61 @@
+/* CREATE TABLES FOR DIAGRAM */
+
+-- patients
+CREATE TABLE patients (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(50) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  PRIMARY KEY(id)
+);
+
+-- medical history
+CREATE TABLE medical_histories (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INT NOT NULL,
+  status VARCHAR(250) NOT NULL,
+  FOREIGN KEY (patient_id) REFERENCES species(id);
```

Comment:
- Hey, I think you meant the `patients(id)` as the `species table` does not exist in the diagram given. please, kindly verify the table name to fix the issue. :pray: